### PR TITLE
commmunity/terraform: upgrade to 0.10.8

### PR DESCRIPTION
**Pull request by Ftornik**

### CODE REVIEW

**Comment by jbergstroem** [comment]
TL;DR: Travis says:
```
runtime: address space conflict: map(0xc420000000) = 0x7fd404eb9000
fatal error: runtime: address space conflict
```

..while building `aws/aws-sdk-go/service/ec2`. Retry?

**Comment by Ftornik** [comment]
I've added some build flags. Build successful.

File: community/terraform/APKBUILD [hunk 1]
```diff
@@ -21,12 +21,13 @@ prepare() {
...
 build() {
 	cd "$builddir"
-	GOPATH="$srcdir" go build -v -o bin/$pkgname
+	GOPATH="$srcdir" go build -v -o bin/$pkgname \
+		-ldflags "-X main.GitCommit=v$pkgver -X github.com/hashicorp/terraform/version.Prerelease= -s -w"
 }
 
 check() {
 	cd "$builddir"
-	GOPATH="$srcdir" go test
```
Comment (jbergstroem) [inline community/terraform/APKBUILD]: Why change this? I thought `check()` is to run the test suite, not "Hello World"?

Comment (Ftornik) [reply community/terraform/APKBUILD]: Fixed

File: community/terraform/APKBUILD [hunk 2]
```diff
@@ -20,12 +21,18 @@ prepare() {
 
 build() {
 	cd "$builddir"
-	export GOPATH="$srcdir"
-	go build -v -o bin/$pkgname
+	GOPATH="$srcdir" go build -v -o bin/$pkgname \
+		-ldflags "-X main.GitCommit=v$pkgver -X github.com/hashicorp/terraform/version.Prerelease= -s -w"
```
Comment (jbergstroem) [inline community/terraform/APKBUILD]: For reviewers; this follows upstream procedures [done in their release builds](https://github.com/hashicorp/terraform/blob/master/scripts/build.sh#L46..L48).

**Comment by ncopa** [comment]
I rebased it and merged the commits. Thanks!

**Pull request closed without merging**